models/public/hrnetv2/accuracy-check.yaml

```diff
@@ -0,0 +1,77 @@
+models:
+
+
+  - name: hrnetv2_pytorch
+
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        model:   public/hrnetv2/hrnetv2.onnx
+        adapter: segmentation
+
+    datasets:
+      - name: ADEChallengeData2016
+        preprocessing:
+          - type: bgr_to_rgb
+          - type: resize
+            size: 320
+            use_pillow: True
+            interpolation: BILINEAR
+          - type: normalization
+            mean: (123.675,116.28,103.53)
+            std: (58.395,57.12,57.375)
+        postprocessing:
+          - type: resize_segmentation_mask
+            apply_to: prediction
+          - type: shift
+            apply_to: annotation
+            shift_value: -1
+        metrics:
+          - name: pix_accuracy
+            type: segmentation_accuracy
+            use_argmax: True
+          - name: mean_IoU
+            type: mean_iou
+            use_argmax: True
+
+  - name: hrnetv2_pytorch
+
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        tags:
+          - FP16
+        model:   public/hrnetv2/FP16/hrnetv2.xml
+        weights: public/hrnetv2/FP16/hrnetv2.bin
+        adapter: segmentation
+
+      - framework: dlsdk
+        device: CPU
+        tags:
+          - FP32
+        model:   public/hrnetv2/FP32/hrnetv2.xml
+        weights: public/hrnetv2/FP32/hrnetv2.bin
+        adapter: segmentation
+ 
+    datasets:
+      - name: ADEChallengeData2016
+        preprocessing:
+          - type: bgr_to_rgb
```

Comment:
why do you need bgr_to_rgb?
is it impossible to include --reverse_input_channels to the model during MO conversion?